nexus_constructor/model/stream.py

```diff
@@ -50,6 +50,11 @@ class EV42Stream:
     topic = attr.ib(type=str)
     source = attr.ib(type=str)
     writer_module = attr.ib(type=str, default=WriterModules.EV42.value, init=False)
+    adc_pulse_debug = attr.ib(type=bool, default=None)
```

Comment:
I think these may need to match the names of the ones in the filewriter JSON, jonas said he wanted to change them to a flat structure so `nexus.indices.index_every_mb` is valid. perhaps we should just use underscores in place of the `.`s

also, i think some of them are booleans rather than ints, so the type should match in the `attr.ibs` declaration. the documentation [here](https://github.com/ess-dmsc/kafka-to-nexus/tree/v3.0.0/documentation) should hopefully help, i can't remember which ones were booleans off the top of my head